css-transitions/styles.css

```diff
@@ -0,0 +1,53 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: sans-serif;
+  background-color: rgb(245, 245, 245);
+}
+
+section {
+  max-width: 992px;
+  margin: 2rem auto;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.custom-button {
+  padding: 0.5rem 1.5rem;
+  border: none;
+  border-radius: 0.25rem;
+  font-size: 1rem;
+  color: white;
+  background-color: blue;
+}
+
+.custom-button:hover {
+  background-color: red;
+  transition: background-color 0.5s;
```

Comment:
As we discussed in Q&A, this should be on the primary rule for the element, so the transitions do not only take effect while the pseudoclass is applied to the element :grin: